src/request_handler.cc

```diff
@@ -3,12 +3,13 @@
 #include <cstring>
 
 RequestHandler* RequestHandler::CreateByName(const char* type) {
-    // TODO(evan): use static registeration
-    std::string typeStr(type, std::strlen(type));
-    if (typeStr == "EchoHandler") {
-        return new EchoHandler();
-    } else {
-        std::cerr << "Unknown handler name '" << typeStr << "'" << std::endl;
+    if ( ! request_handler_builders) {
+        std::cerr << "No Request Handlers registered!" << std::endl;
```

Comment:
not really too big but i've been using \n to end my lines  do you want to stick with endl or \n